contracts/src/v0.8/functions/dev/v1_X/FunctionsBilling.sol

```diff
@@ -96,7 +107,14 @@ abstract contract FunctionsBilling is Routable, IFunctionsBilling {
 
   /// @inheritdoc IFunctionsBilling
   function getDONFee(bytes memory /* requestData */) public view override returns (uint72) {
-    return s_config.donFee;
+    // s_config.donFee is in cents of USD. Get Juel amount then convert to dollars.
+    return SafeCast.toUint72(_getJuelsFromUsd(s_config.donFee) / 100);
+  }
+
+  /// @inheritdoc IFunctionsBilling
+  function getOperationFee() public view override returns (uint72) {
```

Comment:
`getOperationFeeJuels()`

Reply:
I'm a little hesitant to do this in order to keep it in line with getDONFee. I think the natspec documentation should be enough.

Reply:
Changed